### DISCUSSION TITLE
纯属好奇，问下大佬是怎么把windows版压缩到10MB左右的

### DISCUSSION THREAD

**Discussion by kbtxwer** [AlistGo/alist#1506] [Q&A]
我在windows下用 go build 编译的可执行文件体积差不多70M，用了 upx -9 压缩后还有约20M，看脚本大佬似乎也没有再用别的东西压缩了，不知道是怎样把体积弄这么小的

**Comment by xhofe** [comment]
没有什么特殊的技巧，所有的workflow文件都是开源的，编译也是GitHub actions自动构建的。

**Reply by kbtxwer** [reply]
嗯嗯，那以后我再用Linux试试看，不过只要能测试就挺好的，剩下的还有自动构建帮助
原因我找到了，是[用 -w -s 删了调试符号从而大幅度减小了体积](https://www.cnblogs.com/bergus/articles/go-ldflags-she-zhi.html)
![9}49%@(}2X3OLCXQMM7NLES](https://user-images.githubusercontent.com/57350104/185748352-24161023-969c-456b-9449-e50ca4f9f8dc.png)
